src/components/RightSidebar.test.tsx

```diff
@@ -21,6 +22,11 @@ describe("RightSidebar", () => {
     expect(r.rightSidebar_content).toHaveTextContent("Comments");
     expect(r.rightSidebar_content).not.toHaveTextContent("History");
 
+    // wait for transition to finish
+    await waitFor(() => {
```

Comment:
there was a scenario in which the animation was rendering two `time` buttons in the DOM when switching icons, so I added this `waitFor` to ensure we actually only have one